src/test/resources/com/puppycrawl/tools/checkstyle/xpath/xpathquerygenerator/InputXpathQueryGeneratorEscapeCharacters.java

```diff
@@ -0,0 +1,7 @@
+package com.puppycrawl.tools.checkstyle.xpath.xpathquerygenerator;
+
+public class InputXpathQueryGeneratorEscapeCharacters {
+    String testOne = "<>'\"&abc;&lt;\u0080\n";
+
+    String testTwo = "&#0;&#X0\u0001\n";
```

Comment:
Please add test case with text block
New jdk feature where - multi line string.
Better to create new Input file for such sofisticated input. It will have invisible \n

Reply:
Done, but `TEXT_BLOCK_CONTENT` are not suppressed like String, the entire block is suppressed not line and the inside content doesn't matter if there is any violation inside block.